src/Dfe.ContentSupport.Web/Services/LayoutService.cs

```diff
@@ -61,7 +61,7 @@ public List<PageLink> GenerateVerticalNavigation(CsPage page, HttpRequest reques
     }
 
 
-    public static List<CsContentItem> GetVisiblePageList(CsPage page, string pageSlug)
+    public List<CsContentItem> GetVisiblePageList(CsPage page, string pageSlug)
```

Comment:
What was wrong with the static method?

Reply:
[1aad58a](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/811/commits/1aad58a750b7b64bb5a326a9a35751aafb3ffbc6)